IndexedDB/idbindex_keyPath.any.js

```diff
@@ -28,3 +28,49 @@ indexeddb_test(
     t.done();
   },
   `IDBIndex's keyPath attribute returns the same object.`);
+
+  indexeddb_test(
+  (t, db) => {
+    const store = db.createObjectStore('store', {autoIncrement: true});
+    store.createIndex('index', ['a']);
+
+    store.add({a: 1, b: 2, c: 3})
+  },
+  (t, db) => {
+    const tx = db.transaction('store', 'readonly', {durability: 'relaxed'});
+    const store = tx.objectStore('store');
+    const index = store.index('index');
+    const cursorReq = index.openCursor();
+
+    cursorReq.onsuccess = t.step_func((e) => {
+      const expectedKeyValue = [1];
+      const actualKeyValue = e.target.result.key.valueOf();
+
+      assert_array_equals(actualKeyValue, expectedKeyValue);
```

Comment:
Can you add a message to the assertion? Ideally, it explains the assertion to someone that sees the failure, sees the expected/actual values, and wants to understand what is actually being asserted without deciphering the code. A good phrasing includes "should", e.g. _"An array keypath should yield an array key."_